### DISCUSSION TITLE
Copilot api endpoint seems to be unreachable from indonesia origin

### DISCUSSION THREAD

**Discussion by Riskev02** [community/community#141659] [Copilot Conversations]
### Select Topic Area

Question

### Body

is there someone else have same issue? when run the github copilot chat diagnostic from vscode, all of the Fetcher returned by connection timeout currently when try to Connecting on https://api.githubcopilot.com/_ping

**Comment by senndbx** [comment]
I've got the same issue here, swapped between multiple connection and still getting net::ERR_CONNECTION_TIMED_OUT

**Reply by Riskev02** [reply]
yeah, i think there is some issue for indonesia region, i tried to use sg vpn and it resolved the issue

**Comment by Riskev02** [comment]
it seemed github copilot has ben reachable from my device, so i will close this discussion, thanks
